partcad-cli/src/partcad_cli/click/commands/add/sketch.py

```
@@ -0,0 +1,55 @@
+            "dxf",
+            "svg",
+            "basic",
+            "ai-cadquery",
+            "ai-openscad",
+        ]
+    ),
+    # help="Type of the sketch",
+)
+@click.argument("path", type=str)  # help="Path to the file"
+@click.pass_obj
+def cli(ctx, desc, kind, provider, path):
```

Ruff (0.8.2)

Missing return type annotation for public function `cli`

Add return type annotation: `None`

(ANN201)